migrations/0003_auto_20200917_1017.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.3 on 2020-09-17 04:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('maintainer_site', '0002_hit'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='maintainergroup',
+            name='medium_slug',
+            field=models.CharField(blank=True, max_length=63, null=True),
```

Comment:
You forgot to run `python manage.py makemigrations`. Also, make sure you squash the migration generated and this one.